src/domain/file/usecase/remove-all-files/removeAllFiles.ts

```diff
@@ -0,0 +1,12 @@
+import type { FileEntity } from 'domain/file/entity/file'
+import type { ThunkResult } from 'domain/file/store/store'
+import { RemoveFileActions } from '../remove-file/actionCreators'
+
+export const removeAllFiles =
+  (): ThunkResult<Promise<void>> =>
+  // eslint-disable-next-line @typescript-eslint/typedef
+  async (dispatch, getState) => {
+    getState().file.byId.forEach((_value: FileEntity, key: string) =>
```

Comment:
I would have called it `id` rather than `key`, since what is represented is actually the id, but it's up to you.

Reply:
For sure, that is more explicit 👌